relay-event-normalization/src/transactions/processor.rs

```diff
@@ -89,14 +107,7 @@ impl<'r> TransactionsProcessor<'r> {
 
     fn normalize_transaction_name(&self, event: &mut Event) {
         if self.treat_transaction_as_url(event) {
```

Comment:
not sure how to deal with the `treat_transaction_as_url` logic here. It uses data in `transaction_info` field of `Event` which the DSC does not have access to. I only added the 'contains / ' condition from it

Reply:
Yeah I think this is the best we can do for now. The chance that a rule derived from URL transactions accidentally also matches & messes up a non-URL transaction is low, so I think we can risk it.